From f8ecc7fb09adf34a23d324f373e667c81311fc1c Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@gmail.com>
Date: Sat, 6 Aug 2016 23:24:35 +0200
Subject: [PATCH] configure: match uclinux pattern

bdwgc does not recognize "uclinux" as a valid OS part of the target
tuple which is used by some arm cortex-M toolchains.

Fixes:
http://autobuild.buildroot.net/results/94f/94fbc1e5afe183e5b071d1e869b2d780025389e2

Signed-off-by: Romain Naour <romain.naour@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index b1ce2b2..a69d101 100644
--- a/configure.ac
+++ b/configure.ac
@@ -186,7 +186,7 @@ case "$THREADS" in
         AC_DEFINE(THREAD_LOCAL_ALLOC)
         AC_MSG_WARN("Explicit GC_INIT() calls may be required.");
         ;;
-     *-*-linux*)
+     *-*-*linux*)
         AC_DEFINE(GC_LINUX_THREADS)
         AC_DEFINE(_REENTRANT)
         ;;
-- 
2.5.5

